Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -905,7 +905,7 @@ TccModePostMemConfig (
     Status = LoadComponent (SIGNATURE_32 ('I', 'P', 'F', 'W'), SIGNATURE_32 ('T', 'C', 'C', 'T'),
                             (VOID **)&TccStreamBase, &TccStreamSize);
     if (EFI_ERROR (Status) || (TccStreamSize < sizeof (TCC_STREAM_CONFIGURATION))) {
-      DEBUG ((DEBUG_INFO, "Load TCC Stream %r, size = 0x%x\n", Status, TccStreamSize));
+      DEBUG ((DEBUG_ERROR, "Load TCC Stream %r, size = 0x%x\n", Status, TccStreamSize));
```

Comment:
This is an info msg, not error msg.

Reply:
since the debug print goes through the failed status path, so I made it as DEBUG_ERROR;
but I am not quite sure how to distinguish DEBUG_INFO from DEBUG_ERROR